source/route_auth.js

```diff
@@ -12,27 +12,27 @@ angular.module( "vokal.RouteAuth", [ "LocalStorageModule" ] )
         redirectPath = path;
     };
 
-    this.$get = [ "$q", "$location", "$rootScope", "localStorageService",
+    this.$get = [ "$q", "$location", "$rootScope",
 
-        function ( $q, $location, $rootScope, localStorageService )
+        function ( $q, $location, $rootScope )
         {
             var service = {
-
+                storageType: localStorage,
                 loadRoles: function ()
```

Comment:
Not part of this commit, but would it make sense to include a run block for this function to prevent having to check `if( roles === null )` in the later functions?